covidapp/android/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="app_name">covidapp</string>
```

Comment:
Just name it "who-app" right now

Reply:
My one concern with that: Better to rename once WHO has given the sign-off and the contents are medically reviewed.  Don't want ppl thinking our lightning-speed prototypes are gospel.  Do you agree?